tests/test_workflow/test_to_script.py

```diff
@@ -1,100 +1,125 @@
+                                                    ' filename=\'filename\', extension=\'json\', text=True,'
+                                                    ' name="Export", position=(99, 88.88))']
+
+blocks = [block_0, block_1, block_2, block_4, block_5]
+
+# Variables
+variable_list = [[Variable(name='variable_0', position=[-6.77, 34.65], type_=str),
+                  "Variable(name='variable_0', position=[-6.77, 34.65], type_=str)"],
+                 [Variable(name='variable_1', position=[-632.77, 34.65], type_=List[BinaryFile]),
+                  "Variable(name='variable_1', position=[-632.77, 34.65], type_=List[BinaryFile])"],
+                 [Variable(name='variable_2', position=[-2342.61, 18.72], type_=List[MyObj]),
+                  "Variable(name='variable_2', position=[-2342.61, 18.72], type_=List[MyObj])"],
+                 [Variable(name='variable_3', position=[197.91, 95.02], type_=List[int]),
+                  "Variable(name='variable_3', position=[197.91, 95.02], type_=List[int])"],
+                 [Variable(name='variable_4', position=[-1035.71, -269.35], type_=List[str]),
+                  "Variable(name='variable_4', position=[-1035.71, -269.35], type_=List[str])"],
+                 [Variable(name='variable_5', position=[2620.96, -486.85], type_=List[bool]),
+                  "Variable(name='variable_5', position=[2620.96, -486.85], type_=List[bool])"],
+                 [Variable(name='variable_6', position=[2620.96, -486.85], type_=Dict[str, int]),
+                  "Variable(name='variable_6', position=[2620.96, -486.85], type_=Dict[str, int])"],
+                 [Variable(name='variable_7', position=[2620.9, -486.8], type_=List[List[Tuple[float, float]]]),
+                  "Variable(name='variable_7', position=[2620.9, -486.8], type_=List[List[Tuple[float, float]]])"]]
+
+# Pipes
+pipe_1 = Pipe(blocks[0][0].outputs[0], blocks[1][0].inputs[0])
+pipe_2 = Pipe(variable_list[1][0], blocks[0][0].inputs[2])
+pipe_3 = Pipe(variable_list[2][0], blocks[0][0].inputs[1])
+pipe_4 = Pipe(variable_list[3][0], blocks[0][0].inputs[3])
+pipe_5 = Pipe(variable_list[4][0], blocks[0][0].inputs[5])
+pipe_6 = Pipe(variable_list[5][0], blocks[0][0].inputs[4])
+pipe_7 = Pipe(variable_list[0][0], blocks[0][0].inputs[0])
+pipe_8 = Pipe(variable_list[6][0], blocks[0][0].inputs[6])
+pipe_9 = Pipe(variable_list[7][0], blocks[0][0].inputs[7])
+pipe_10 = Pipe(blocks[1][0].outputs[0], blocks[2][0].inputs[0])
+pipe_11 = Pipe(blocks[1][0].outputs[0], blocks[3][0].inputs[0])
+
+workflow = Workflow([block[0] for block in blocks],
+                    [
+                        pipe_1, pipe_2, pipe_3, pipe_4, pipe_5, pipe_6, pipe_7,
+                        pipe_8, pipe_9, pipe_10, pipe_11
+                    ],
+                    output=blocks[1][0].outputs[0], name="TestExport")
+
+
+class WorkflowToScriptTest(unittest.TestCase):
+
+    def test_blocks(self):
+        for block in blocks:
```

Comment:
You could use parameterized. Please check other test files that use it

Reply:
done